docker/java/spring/Dockerfile

```diff
@@ -19,16 +19,14 @@ RUN cd /agent/apm-agent-java \
 COPY maven-package.sh /agent
 RUN ./maven-package.sh "${JAVA_AGENT_BUILT_VERSION}"
 
-FROM openjdk:10-jre-slim
+FROM adoptopenjdk:11-jre-hotspot
```

Comment:
I guess this Java version bump is intended, right?

Reply:
yep, let's use the same we use in master